django/db/migrations/state.py

```diff
@@ -443,6 +443,11 @@ def from_model(cls, model, exclude_rels=False):
                 elif name == "index_together":
                     it = model._meta.original_attrs["index_together"]
                     options[name] = set(normalize_together(it))
+                elif name == "indexes":
+                    indexes = model._meta.indexes
```

Comment:
That's absolutely right.

I'm wondering if it might be helpful to implement `Index.clone()` which does a deconstruct/reconstruct  a la https://github.com/django/django/blob/master/django/db/models/fields/__init__.py#L463-L469